src/dune_pkg/feature_flags.mli

```diff
@@ -0,0 +1,27 @@
+open! Import
+
+(** Ad-hoc feature flags for enabling various features for the dune
```

Comment:
The only comments with the possibility of showing up in external documentation (e.g. web pages generated by the code) are those wrapped in `(** ... *)` (not `(* ... *)`). However this module is private to dune rather than a library that gets released publicly, so these comments won't be published on any websites - they are just going to show up in people's editors when querying the documentation of these flags.